### PR TITLE
[tests] Added POI priority tests.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

**Comment by mpimenov** [comment]
LGTM

File: map/style_tests/classificator_tests.cpp [hunk 1]
```diff
@@ -151,69 +151,102 @@ pair<int, int> GetMinMax(int level, vector<uint32_t> const & types)
...
+    vector<StringIL> arrT =
+    {
+      // 1
+      {"amenity", "bench"}, {"amenity", "shelter"},
+      // 2
+      {"highway", "bus_stop"}, {"amenity", "bus_station"},
+      {"railway", "station"}, {"railway", "halt"}, {"railway", "tram_stop"},
+    };
+
+    CheckPriority(arrT, { 2, 5 }, drule::symbol);
```
Comment (ygorshenin) [inline map/style_tests/classificator_tests.cpp]: clang-format please

File: map/style_tests/classificator_tests.cpp [hunk 2]
```diff
@@ -151,69 +151,102 @@ pair<int, int> GetMinMax(int level, vector<uint32_t> const & types)
...
+    {"natural", "water"}, {"natural", "lake"}, {"landuse", "basin"}
+  };
+
+  CheckPriority(arrT, { 1, 2, 8, 3 }, drule::area);
+}
+
+UNIT_TEST(Classificator_PoiPriority)
+{
+  {
+    vector<StringIL> arrT =
```
Comment (ygorshenin) [inline map/style_tests/classificator_tests.cpp]: s/arrT/types/g
?

**Comment by vng** [comment]
PTAL

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**